tests/lib/eslint-tester.js

```diff
@@ -329,6 +329,27 @@ describe("ESLintTester", function() {
         });
     });
 
+    it("should pass-through the parser to the rule", function() {
```

Comment:
dev dependencies are cheap, feel free to add. 

It's probably easiest to expose the parser being used on ESLintTester. Maybe `getParser()`? And then compare it to what you get from `require()`